doc/index.hbs

```diff
@@ -12,3 +12,7 @@ For a more in-depth overview of OpenLayers 3 core concepts, check out the [tutor
 Make sure to also check out the [OpenLayers 3 workshop](../../../ol3-workshop/).
 
 Find additional reference material in the [API docs](../apidoc).
+
+# Questions
+
+If you cannot find an answer in the documentation, you can ask your question on [stackoverflow using the tag 'openlayers-3'](http://stackoverflow.com/questions/tagged/openlayers-3).
```

Comment:
The `openlayers` tag should be used, see Eric's comment: https://github.com/openlayers/ol3/commit/2ff3682eff4cfbf34297dcaade8b853f2aac2a56#commitcomment-9986570